core/templates/cowdata.h

```diff
@@ -436,20 +436,19 @@ Error CowData<T>::_realloc(Size p_alloc_size) {
 
 template <typename T>
 typename CowData<T>::Size CowData<T>::find(const T &p_val, Size p_from) const {
-	Size ret = -1;
```

Comment:
The use of `ret` here should most likely be optimized out, but it's cleaner this way and also matches `rfind`